frontend/src/metabase/components/DownloadButton.jsx

```diff
@@ -0,0 +1,41 @@
+import React, { Component, PropTypes } from "react";
+import ReactDOM from "react-dom";
+
+import Button from "metabase/components/Button.jsx";
+
+export default class DownloadButton extends Component {
+    constructor(props, context) {
+        super(props, context);
```

Comment:
There's no need to pass context in here since we're not using context. (I think a ton of our components have this copypasta problem).